addons/l10n_ch/models/account_invoice.py

```diff
@@ -40,8 +40,11 @@ class AccountMove(models.Model):
     @api.depends('partner_id', 'currency_id')
     def _compute_l10n_ch_qr_is_valid(self):
         for move in self:
-            move.l10n_ch_is_qr_valid = move.move_type == 'out_invoice' \
-                                       and move.partner_bank_id._eligible_for_qr_code('ch_qr', move.partner_id, move.currency_id, raises_error=False)
+            move.l10n_ch_is_qr_valid = (
+                move.move_type == 'out_invoice'
+                and move.partner_bank_id._eligible_for_qr_code('ch_qr', move.partner_id, move.currency_id, raises_error=False)
+                and move.company_id.qr_code
+            )
```

Comment:
From what I see:

`_compute_l10n_ch_qr_is_valid` should be an override of  `display_qr_code` specialized for invoices with `move.qr_code_method == 'ch_qr'`.

References:
https://github.com/odoo/odoo/tree/16.0/addons/account/models/account_move.py#L1488
https://github.com/odoo/odoo/tree/16.0/addons/account/models/account_move.py#L4147

Can you verify please? ^^